NMS-14850: Review enlinkd documentation (#5499)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/topology/enlinkd/layer-3-discovery.adoc

[[ga-enlinkd-layer-3-link-discovery]]
= Layer 3 Link Discovery

Using enlinkd, it is possible to get links based on data from network routing applications.

> The following routing daemons can be used to provide links-based layer 3 information:

Reviewer: ```suggestion
You can use the following routing daemons to provide links-based Layer 3 information:
```

* link:https://en.wikipedia.org/wiki/Open_Shortest_Path_First[Open Shortest Path First] (OSPF)
* link:https://en.wikipedia.org/wiki/IS-IS[Intermediate System to Intermediate System] (IS-IS)

> Information on level 2 network links is provided by SNMP agents that support the appropriate MIBs.

Reviewer: ```suggestion
SNMP agents that support the appropriate MIBs, provide information on Layer 2 network links.
```

> For this reason, you are required to have a working SNMP configuration.

Reviewer: ```suggestion
For this reason, you must have a working SNMP configuration.
```

> The link data that is discovered by enlinkd is detailed in the topology user interface, and on a node's Details page.

Reviewer: ```suggestion
The link data that enlinkd discovers is detailed in the topology user interface, and on a node's details page.
```
